### ISSUE TITLE
Outgoing typing notifications prompt an error reply

### ISSUE THREAD

**Issue by moppman** [dino/dino#607]
Hi there,
I noticed that outgoing typing notifications from Dino don't show up in other clients.

Dino version: 0.0~git523.9f613d5
Typing notifications globally enabled.
From Dino's  debug log (via dino --print-xmpp=all)
```
XMPP OUT [dino@own.server]
<message id='a94da5d0-58fd-46f0-bbdc-5d296e1be86e' to='contact@other.server' type='groupchat'>
  <paused xmlns='http://jabber.org/protocol/chatstates' />
  <no-store xmlns='urn:xmpp:hints' />
</message>

XMPP IN [dino@own.server]
<message type='error' to='dino@own.server/dino.f77402f9' from='contact@other.server' id='a94da5d0-58fd-46f0-bbdc-5d296e1be86e' lang='en'>
  <error type='cancel'>
    <service-unavailable xmlns='urn:ietf:params:xml:ns:xmpp-stanzas' />
  </error>
</message>
```
I'm not sure why the outgoing stanza is of type groupchat, because this is a one on one chat.
Thanks!

**Comment by fiaxh** [comment]
Thanks for opening an issue. You are right about the problem, the type should be `chat` instead of `groupchat`. I pushed a fix: 016ab2c.

**Comment by moppman** [comment]
I just upgraded and https://github.com/dino/dino/commit/016ab2c19b0760792f3ea8f00bfc46b49500f96a fixes it.
Thanks!
